aws-frauddetector-eventtype/src/aws_frauddetector_eventtype/helpers/api_helpers.py

```diff
@@ -110,6 +110,49 @@ def collect_items_of_interest_from_current_response():
     return paginated_api_call_decorator
 
 
+def batch_call_limit(request_attribute_to_split: str, response_items_to_collect: List[str], limit: int = 100):
```

Comment:
This decorator lets callers of annotated methods not worry about splitting up batch calls into requests that the API can handle. BatchCreateVariable has a limit of 25, where BatchGetVariable has a limit of 100.